.github/workflows/test-action.yaml

```diff
@@ -0,0 +1,26 @@
+name: Tests
```

Comment:
let's rename to something like `Run checks`

Reply:
`+` Java Build and Test Workflow

Reply:
But you haven't java in the project :))

Reply:
`+`